src/beacon-api/app/api/v1/webrtc/participant_controller.py

```
@@ -0,0 +1,55 @@
+from app.api.v1.operations import get_current_user_id
+from app.schemas.webrtc import WebRTCSessionMemberResponse, GrantControlRequest, RevokeControlRequest
+from app.services.webrtc.participant_service import participant_service
+from app.services.webrtc.permission_service import permission_service
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+@router.get("/sessions/{id}/participants", response_model=List[WebRTCSessionMemberResponse])
+async def list_participants(
+    id: UUID,
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)